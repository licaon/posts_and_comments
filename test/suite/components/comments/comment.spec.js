import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Comment from 'src/components/comments/comment';

describe('Comment component', () => {
  const createComponent = (props) => {
    const wrapper = shallow(
      <Comment
        {...props}
      />);
    return toJson(wrapper);
  };
  it('should render component correctly with props', (done) => {
    const component = createComponent({
      comment: {
        name: 'name',
        body: 'body',
      },
    });

    expect(component).toMatchSnapshot();
    done();
  });
});
