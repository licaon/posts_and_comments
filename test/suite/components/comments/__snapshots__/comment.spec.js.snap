// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comment component should render component correctly with props 1`] = `
<Card
  className="comment"
  expanderIconChildren="keyboard_arrow_down"
  expanderIconClassName="material-icons"
  expanderTooltipPosition="left"
>
  <CardTitle
    title="name"
  />
  <CardText
    component="section"
  >
    body
  </CardText>
</Card>
`;
